lib/piper/command/ast/pipeline.ex

```diff
@@ -1,10 +1,12 @@
 defmodule Piper.Command.Ast.Pipeline do
 
+  require Piper.Command.Ast.PipelineStage
```

Comment:
Well, I thought this was necessary to ensure `PipelineStage` was compiled and loaded before `Pipeline` but now that I look at the docs this is only necessary when the caller is using macros from the `require`d module. I'll fix this up.